proposals/pytorch-operator-proposal.md

```diff
@@ -0,0 +1,141 @@
+## Motivation
+Pytorch is a popular machine learning framework which currently does not have an operator/controller for Kubernetes. This proposal is aimed at defining what that operator should look like, and adding it to Kubeflow.
```

Comment:
should we use `PyTorch` instea of `Pytorch`?

Reply:
Yeah I'll update